crates/modelardb_server/src/storage/mod.rs

```diff
@@ -228,13 +228,22 @@ impl StorageEngine {
             )?;
         }
 
-        Ok(Self {
+        let mut storage_engine = Self {
             uncompressed_data_manager,
             compressed_data_manager,
             memory_pool,
             join_handles,
             channels,
-        })
+        };
+
+        // Start the task that transfers data periodically if a data transfer component exists and
+        // time-based data transfer is enabled. Errors are ignored since an error is only returned
+        // if a data transfer component does not exist.
+        let _ = storage_engine
```

Comment:
Maybe check if the data transfer component is enabled before calling the method and return all errors returned by `set_transfer_time_in_seconds(...)` as ignoring all errors may create issues that are hard to debug in the future, e.g., if we start returning additional errors which are then also ignored which we will have forgotten about at that time.

Reply:
Good idea, fixed.